### PR TITLE
Feature/add community tabs overlay

### PR DESCRIPTION
**Pull request by courtneymyers**
## Related Issues:
* https://app.breeze.pm/projects/100762/cards/3185187

## Main Changes:
* Adds a semi-transparent gradient overlay over the Community Tabs (persistent over the first and last tab shown in the viewport)

## Steps To Test:
1. Visit a specific community page (e.g. /community/22201/overview) and view the community tabs

### CODE REVIEW

**Comment by courtneymyers** [comment]
> Looks great @courtneymyers not seeing any issues! I like the 3D effect it adds

Thanks, @oftheheadland!

File: app/client/src/components/pages/Community/components/routes/CommunityTabs.js [hunk 1]
```diff
@@ -90,7 +156,7 @@ const StyledTabs = styled(Tabs)`
       &[data-selected],
       &:hover,
       &:focus {
-        z-index: 1;
+        z-index: 1
```
Comment (cschwinderg) [inline app/client/src/components/pages/Community/components/routes/CommunityTabs.js:159]: The semicolon got removed, which is taking away the background color for the selected tab. Can you add that back in?

```suggestion
        z-index: 1;
```

**Comment by courtneymyers** [comment]
> While I was testing I was thinking that a possible enhancement (no need to change anything now) would be to only show the overlay on the side(s) that has tabs that aren't visible. So, when you are on the overview tab the overlay is only displayed on the right side. Then when you scroll to the right the overlay shows up on both sides, until you get to the last tab at which point the overlay is only on the left.

Ha! That's the exact thing I mentioned to Brad when we met...I agree that'd be a nice touch, and mentioned to him I tried it, but found it too difficult to implement cleanly. We can revisit it as a possible future enhancement. I like the way you think!

File: app/client/src/components/pages/Community/components/routes/CommunityTabs.js (hunk 1 above)
Comment (courtneymyers) [reply app/client/src/components/pages/Community/components/routes/CommunityTabs.js:159]: Thank you for catching this!

**Pull request merged**